src/viking.js

```diff
@@ -1,11 +1,52 @@
 // Soldier
-class Soldier {}
+ class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack(){
+     return this.strength;
+  }
+  receiveDamage(damage){
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor (name,health, strength){
+        super(health, strength);
+       this.name = name; 
+    }
+     receiveDamage (damage){
+        super.receiveDamage (damage);
+        if (this.health > 0){
+            return `${this.name} has received ${damage} points of damage`;
+        }else{
+            return `${this.name} has died in act of combat`;
+        }
+     }
+     battleCry (){
+        return `Odin Owns You All!`;
+     }
+}
+
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+    constructor (health, strength){
+        super (health, strength);
+        }
+        receiveDamage(damage){
+            super.receiveDamage(damage);
+            if (this.health > 0){
+                return `A Saxon has recived ${damage} points of damage`;
```

Comment:
You have a typo here that is making the error on jasmine.

A Saxon has `received` ${damage} points of damage;